docs/how-to/installation/kubernetes.txt

```diff
@@ -64,11 +64,11 @@ Number of GPUs Per Node
 #######################
 
 Users are required to specify the number of GPUs on each node (for GPU-enabled nodes only). This is done
-by setting ``slotsPerNode`` in ``values.yaml``. Determined uses this information when scheduling multi-GPU tasks.
-Each multi-GPU (distributed training) task will be scheduled as a set of ``slotsPerTask / slotsPerNode`` separate pods,
-with each pod assigned to ``slotsPerNode`` GPUs. Tasks with sizes that are not divisible by ``slotsPerNode`` are never scheduled.
-If you have a cluster of different size nodes set ``slotsPerNode`` to the smallest common denominator. For example, if
-you have nodes with 4 GPUs and other nodes with 8 GPUs, set ``slotsPerNode`` to 4 so that all distributed experiments
+by setting ``maxSlotsPerPod`` in ``values.yaml``. Determined uses this information when scheduling multi-GPU tasks.
+Each multi-GPU (distributed training) task will be scheduled as a set of ``slotsPerTask / maxSlotsPerPod`` separate pods,
+with each pod assigned up to ``maxSlotsPerPod`` GPUs. Tasks with sizes that are not divisible by ``maxSlotsPerPod`` are never scheduled.
```

Comment:
> Tasks with sizes that are not divisible by ``maxSlotsPerPod`` are never scheduled.

Maybe not worth making this more complicated than it already is, but would that incorrectly imply that tasks whose size is smaller than `maxSlotsPerPod` won't be scheduled?

Reply:
How about:

```
Distributed tasks with sizes that are not divisible by ``maxSlotsPerPod`` are never scheduled.
```

Reply:
sgtm